TransmartCoreGrailsPlugin.groovy

```diff
@@ -95,13 +95,13 @@ A runtime dependency for tranSMART that implements the Core API
             conceptsResourceService(DefaultConceptsResource)
         }
 
-        context.'component-scan'('base-package': 'org.transmartproject.db.dataquery.highdim') {
+        context.'component-scan'('base-package': 'org.transmartproject.db') {
             context.'include-filter'(
                     type:       'assignable',
                     expression: AbstractHighDimensionDataTypeModule.canonicalName)
         }
 
-        context.'component-scan'('base-package': 'org.transmartproject.db.dataquery.highdim') {
+        context.'component-scan'('base-package': 'org.transmartproject.db') {
```

Comment:
If you're casting the net wider you should also remove the explicit bean definitions that won't be needed anymore like `AccessControlChecks`, `InnerClinicalTabularResultFactory`, ...